README.md

```diff
@@ -1,88 +1,86 @@
 # Docker Development Environment
```

Comment:
Maybe easier to simply review at the new file as there is a lot of noise here in the diff. https://github.com/evidence-dev/docker-devenv/blob/ce6bca1bc954c4aad697567e87b279e412b0be1f/README.md